### PR TITLE
Pop up projects info

### PR DESCRIPTION
**Pull request by juanmanuelbyc**
The main objective of this feature is to implement the following interactions:

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.
- Implement popups for both mobile and desktop screen sizes:
Mobile popup

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -711,44 +717,23 @@ ul {
...
+  .cimage {
+    width: 55%;
+    height: 47vh;
+    margin: 5px auto 15px auto;
+    align-self: center;
+    background-size: cover;
     background-repeat: no-repeat;
     background-position: center;
-    transition: all 0.5s ease-out;
   }
```
Comment (1GORDON) [inline css/styles.css]: Hello team ✋

- [ ] You may kindly fix the image on the desktop popup, as it appears stretched compared to the one provided in the Figma design.
- [ ]  You may also position the image properly as it seem to have wider margins on the left and right.
- Your design
Your image

- Figma design
Figma image

File: css/styles.css [hunk 2]
```diff
@@ -362,24 +372,52 @@ ul {
...
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+}
+
+.btns {
+  width: 100%;
+  display: flex;
+  gap: 15px;
+}
```
Comment (1GORDON) [inline css/styles.css:391]: Great job team 🤝

- [ ] You may kindly fix the overflow on your mobile popup. The buttons seem to overflow as the screen size reduces.
Overflow mobile

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @juanmanuelbyc & @Bateyjosue ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Professional Readme ✔️
- Linters are passing ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] You may please add a description to your pull request. This is where you explain or list what you accomplished within the project.
description


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 1 above)
Comment (juanmanuelbyc) [reply css/styles.css]: Thanks for the feedback. its done!

File: css/styles.css (hunk 2 above)
Comment (juanmanuelbyc) [reply css/styles.css:391]: image

Could you please try a more realistic screen relationship width x height to evaluate it? or try setting a commercial device when inspecting.

On my side, it looks good

Comment (HAMMAS-SALEEM) [inline css/styles.css:391]: ## Your design in different scenarios

### Scenario 1 
mobile-design (copy)
### Scenario 2
mobile-view
### Figma Template 
figma

- [ ] You can see that your mobile modal is behaving differently on different screens. Kindly fix this issue :blush:
You can reach out to me on slack **Hammas Bin Saleem** so we can fix this issue in one call :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

### Hey @juanmanuelbyc :wave:

### Highlights :1st_place_medal:
 
:heavy_check_mark: Following HTML, CSS best practices
:heavy_check_mark: Following Project Requirements
:heavy_check_mark: Descriptive PR title
:heavy_check_mark: Descriptive branch name

Good job on implementing some of the changes requested by the previous reviewer, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:

Hey  @juanmanuelbyc :wave:

Great work on implementing the requested changes. Your project is complete. There is nothing else to say about your project. :+1:
Go ahead and merge it. :muscle:

## Highlights :1st_place_medal:
:heavy_check_mark: Following Github Flow
:heavy_check_mark: Following Linters
:heavy_check_mark: Popup is working as expected

### [Optional]

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**